src/server/mod.rs

```diff
@@ -993,13 +1004,53 @@ impl LanguageServer for LspServer {
                     _ => None,
                 });
             if let Some(typ) = hover_type {
+                let supports_markdown = match self
+                    .client_capabilities
+                    .read()
+                {
+                    Ok(client_capabilities) => {
+                        if let Some(text_document) =
+                            (*client_capabilities)
+                                .text_document
+                                .as_ref()
+                        {
+                            if let Some(hover) =
+                                text_document.hover.as_ref()
+                            {
+                                hover.content_format.as_ref().map_or(false, |formats| formats.contains(&lsp::MarkupKind::Markdown))
+                            } else {
+                                false
+                            }
+                        } else {
+                            false
+                        }
+                    }
+                    Err(err) => {
+                        log::error!("{}", err);
+                        false
+                    }
+                };
+                let hover_contents: lsp::HoverContents =
+                    match supports_markdown {
+                        true => lsp::HoverContents::Markup(
+                            lsp::MarkupContent {
+                                kind: lsp::MarkupKind::Markdown,
+                                value: format!(
+                                    "```flux\ntype: {}\n```",
+                                    typ
+                                ),
+                            },
+                        ),
+                        false => lsp::HoverContents::Scalar(
+                            lsp::MarkedString::String(format!(
+                                "type: {}",
```

Comment:
```suggestion
                                "{}",
```
To be consistent with the markdown